app/src/main/java/com/globant/example/mentorapp/home/presentation/view/adapter/ListUsersAdapter.java

```diff
@@ -25,25 +26,25 @@
 
     private List<ModelUserEntity> users;
     private Context context;
-    private BaseRecyclerViewAdapter.onUserClick listener;
+    private ListUsersViewFragment fragmentListener;
 
-    public ListUsersAdapter(List<ModelUserEntity> users, BaseRecyclerViewAdapter.onUserClick listener) {
+    public ListUsersAdapter(List<ModelUserEntity> users) {
         this.users = users;
-        this.listener = listener;
     }
 
     @Override
     public ViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
         View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.list_users_adapter, null);
         context = parent.getContext();
+        fragmentListener = (ListUsersViewFragment) ((HomeScreenActivity) context).getSupportFragmentManager().findFragmentByTag(ListUsersViewFragment.LIST_TAG);
         return new ViewHolder(view);
     }
 
     @Override
     public void onBindViewHolder(ViewHolder holder, int position) {
-        ModelUserEntity user = users.get(position);
-        holder.userNameTextView.setText(user.getName());
-        Picasso.with(context).load(user.getImageUrl())
+        holder.user = users.get(position);
+        holder.userNameTextView.setText(holder.user.getName());
```

Comment:
This is a bit confusing. You can have a setUser method in your view holder and set the data in the view components there.